src/styles/_catalog.scss

```diff
@@ -1,19 +1,35 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
 
-  max-width: 944px;
+$card-width: 200px;
+
+.catalog {
+  gap: 48px;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, $card-width);
```

Comment:
There should be four columns starting at 1024px
![Screenshot from 2023-03-06 12-56-16](https://user-images.githubusercontent.com/33029864/223091487-00c216d9-a5b2-460f-91d0-bb3d35217e0d.png)